app/components/Navbar.tsx

```
@@ -1,15 +1,33 @@
+            <p className='text-2xl font-bold text-gradient hover:scale-105 transition-transform'>RESUME ANALYZER</p>
         </Link>
+        <div className="flex items-center gap-6">
+          {auth.isAuthenticated ? (
+            <>
+              <Link to="/upload" className='primary-button w-fit hover:scale-105 transition-transform'>
+                Upload Resume
+              </Link>
+              <Link to="/wipe" className='primary-button w-fit hover:scale-105 transition-transform'>
+                Wipe Data
+              </Link>
+              <button onClick={auth.signOut} className='primary-button w-fit hover:scale-105 transition-transform'>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)